include/zephyr/drivers/can.h

```diff
@@ -145,6 +145,11 @@ enum can_state {
 /** Frame uses CAN-FD Baud Rate Switch (BRS). Only valid in combination with ``CAN_FRAME_FDF``. */
 #define CAN_FRAME_BRS BIT(3)
 
+/** CAN-FD Error State Indicator (ESI). Indicates that the transmitting node is in error-passive
+ * state. Only valid in combination with ``CAN_FRAME_FDF``.
+ */
+#define CAN_FRAME_ESI BIT(3)
```

Comment:
```suggestion
#define CAN_FRAME_ESI BIT(4)
```

`BIT(3)` is already used for `CAN_FRAME_BRS`.

Reply:
Wow, thanks! I know I had this wrong when I first wrote it, but I was sure I corrected _and_ committed it before testing. Odd. Anyways, fixed.